layers/+lang/scala/config.el

```diff
@@ -29,3 +29,6 @@
 
 (defvar scala-auto-start-backend nil
   "If non nil then ensime/metals will be started when a scala file is opened.")
+
+(defvar scala-auto-treeview nil
```

Comment:
Default setting was "t" before now it's, nil please make it true by default.

Reply:
good point